.rubocop.yml

```diff
@@ -302,7 +302,7 @@ Style/EmptyLiteral:
 Metrics/LineLength:
   Enabled: False
 
-Style/MethodCallParentheses:
+Style/MethodCallWithoutArgsParentheses:
```

Comment:
@jyaworski  no, its already in modulesync_config - https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.rubocop.yml#L310